spec/features/event_image_spec.rb

```diff
@@ -19,20 +18,20 @@
     it "should correctly set an image even if first validation fails" do
       fill_in "Maximale Teilnehmerzahl", :with => nil
       attach_file "event[custom_image]", File.join(Rails.root + 'spec/testfiles/image_upload_test.png')
-      click_button I18n.t(".events.form.create")
+      click_button "create"
 
       fill_in "Maximale Teilnehmerzahl", :with => 25
-      click_button I18n.t(".events.form.create")
+      click_button "create"
       expect(Event.last.image).to be_present
     end
 
     it "should not change the image if validation for a new image fails" do
-      click_button I18n.t(".events.form.create")
+      click_button "create"
       previous_image = Event.last.image
 
       visit edit_event_path(Event.last)
       attach_file "event[custom_image]", File.join(Rails.root + 'spec/testfiles/too_small_image.png')
-      click_button I18n.t('.events.form.update')
+      click_button "update"
```

Comment:
That's what I meant about button conventions in tests. You can use either the button label to click the button, which is the I18n line, or the button id, which is "update".
We mostly use the labels currently, but I used the ids in some places, because the labels got quite long.